docker/ubuntu-sdk/15.04-armhf/Dockerfile

```diff
@@ -34,8 +34,18 @@ RUN apt-get -y --force-yes --no-install-recommends install \
     qt5-doc \
     language-pack-en \
     click
+# go dependencies
+RUN apt-get -y --force-yes --no-install-recommends install \
```

Comment:
Go ahead and combine this with the previous line, not need to have the packages installed in separate lines.

Reply:
Changed it